incubator/mariadb-cluster/templates/master-deploy.yaml

```diff
@@ -86,7 +88,12 @@ spec:
 {{ toYaml .Values.master.resources | indent 10 }}
         volumeMounts:
         - name: data
-          mountPath: /bitnami
+          mountPath: /bitnami/mariadb
+{{- if .Values.master.config }}
```

Comment:
IMO the default configuration should be listed in the `values.yaml` file and this condition should be removed

Reply:
agree

Reply:
Done